docker.d/init.sh

```diff
@@ -79,7 +79,8 @@ export ARTIFACT_UPLOAD_TIMEOUT=1200
 export CONFIG_DIR=/app/configs
 export CONFIG_LOADER=/app/bin/configloader
 export ED25519_PRIVKEY_PATH=$CONFIG_DIR/ed25519_privkey
-export GITHUB_OAUTH_TOKEN=
```

Comment:
This was causing https://github.com/mozilla-releng/scriptworker-scripts/blob/master/docker.d/scriptworker.yml#L23 to always be empty.